Feat: 프론트 작업 & api변경, merge 해결

LeeJaejun98 opened the pull request:
## ✨ 작업 내용

> Thymeleaf 이용한 프론트 작업 및 GetMapping 변경

## 💬 리뷰 요구사항(선택)

> Redis에러로 인해 데이터 전송이 되는지 확인하지 못했습니다... 가능하신분 계시면 검증 해주시면 감사할것 같습니다..

LeeJaejun98 committed:
Fix : thyemleaf 사용 위한 get방식 api변경

LeeJaejun98 committed:
Fix : thyemleaf 사용 위한 get방식 api변경

LeeJaejun98 committed:
Feat: 프론트엔드 페이지 1차 생성

LeeJaejun98 committed:
Merge branch 'develop' of https://github.com/MogakCote/mogak-codingtest into develop

LeeJaejun98 committed:
Fix: import annotation.*

LeeJaejun98 committed:
Fix: 불필요한 주석 삭제

LeeJaejun98 committed:
Fix: adminPage.html로 return 값 변경

LeeJaejun98 committed:
Merge branch 'MogakCote:develop' into develop

LeeJaejun98 committed:
Feat: front-end구현

LeeJaejun98 committed:
Fix: api변경

LeeJaejun98 committed:
Merge remote-tracking branch 'origin/develop' into develop

BaekEuiHun committed:
Feature : 회원가입, 마이프로필, 마이프로필 수정 UI (#73)

* 코드수정

* url 수정

* import 수정

* 이메일 찾기 기능 코드 추가

* 패스워드 변경 요청 기능 생성

* 회원탈퇴 요청 기능 추가

* 마이프로필 수정 요청

* 마이프로필 수정 기능 구현

* 마이프로필 html

* 마이프로필 수정 html

* 회원가입 html

* 이메일 업데이트 관련 로직 추가

* 회원정보 변경 로직 추가

* 회원정보 변경 로직 추가

---------

Co-authored-by: Haneol Oh <otyvs1999@gmail.com>

ohahsis committed:
fix: merge lose resolve

ohahsis committed:
fix: ci-cd script

ohahsis committed:
fix: ci-cd script

LeeJaejun98 committed:
Fix: 사용자 전체 리스트 가져오기

LeeJaejun98 committed:
Fix: 관리자 페이지 js추가

ohahsis committed:
fix: delete unused file

ohahsis committed:
fix: ci-cd script

ohahsis committed:
Update ci-cd.yml

LeeJaejun98 committed:
Fix: merge

LeeJaejun98 committed:
Fix: 관리자페이지 html

ohahsis committed:
fix: ci-cd script

ohahsis committed:
remove file for server

ohahsis committed:
fix: remove redis repositories

ohahsis committed:
fix: add health check auth

LeeJaejun98 committed:
Fix: merge 해결

LeeJaejun98 committed:
Fix: merge

LeeJaejun98 committed:
Fix: merge

LeeJaejun98 committed:
Fix: merge

LeeJaejun98 committed:
Fix: front 코드 추가

LeeJaejun98 committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	.github/workflows/ci-cd.yml

LeeJaejun98 committed:
Fix: front 코드 추가

LeeJaejun98 committed:
Fix: front 코드 추가

LeeJaejun98 committed:
Fix: front 코드 추가

barabamBam commented on src/main/java/com/ormi/mogakcote/notice/presentaion/NoticeController.java:
```diff
@@ -53,6 +57,16 @@ public ResponseEntity<?> updateNotice(
+        if (notice == null) {
+            return "redirect:/api/v1/admin";
+        }
+        model.addAttribute("notice", notice);
```
notice말고 noticeResponse 리턴 해줘야 할 거 같아요!

barabamBam commented on src/main/java/com/ormi/mogakcote/notice/presentaion/NoticeController.java:
```diff
@@ -62,9 +76,27 @@ public ResponseEntity<?> deleteNotice(
+//        return ResponseDto.ok(response);
+//    }
+
+    @GetMapping("/latest5List")
```
관리자 페이지에서도 공지사항 5개만 볼 수 있는 건가요?
전체를 다 확인하는 게 낫지 않을까요?

LeeJaejun98 replied:
이부분은 애초에 공지사항 최신 5개만 보여주기로 애초에 얘기 된거 아니였나요?

barabamBam replied:
그게 사용자한테 아닌가요? 관리자가 기존 자신이 작성한 공지사항을 보려면 방법이 없지 않나요?

barabamBam reviewed (commented):
재준님 수정하시느라 고생하셨습니다!
현재 코드 확인해보면 admin/adminPage로 데이터를 넘기는 메서드가 여러개 존재하는데 이 경우는 Mapping이 api 주소가 아닌 저희가 검색을 통해 직접 접속할 주소를 입력하셔야하는 걸로 알고 있습니다. 저도 확인을 해보지는 못했지만 아마 오류가 발생할 거 같습니다.

LeeJaejun98 committed:
Fix: showEditForm return 값 변경

LeeJaejun98 committed:
Fix: UserAuthResponse에서 User로 변경

LeeJaejun98 committed:
Fix: UserList 삭제

LeeJaejun98 committed:
fix : merge

LeeJaejun98 committed:
fix : merge

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

LeeJaejun98 committed:
Update README.md

tjtjdnjswhd committed:
fix: Redis 설정 수정, PostController URL 중복, 사용하지 않는 config 제거

- application-dev.yml, application-prod.yml: spring.cache -> spring.data.redis 사용. 단 이 방식 사용 시 localhost 안됨
- PostController URL 중복 제거
- RedisConfig 에서 RedisClient 빈 등록 시 host, port, username, password 추가
- InMemoryRedisConfig 제거

LeeJaejun98 committed:
Merge pull request #1 from tjtjdnjswhd/develop

fix: Redis 설정 수정, PostController URL 중복, 사용하지 않는 config 제거

LeeJaejun98 committed:
Merge branch 'develop' of https://github.com/MogakCote/mogak-codingtest into develop

LeeJaejun98 committed:
fix : adminPage로 가는 get을 admin에 생성

LeeJaejun98 committed:
fix : getAll() 생성

LeeJaejun98 committed:
fix : null 삭제

LeeJaejun98 committed:
fix : front 데이터 변경

LeeJaejun98 committed:
merge: pull

LeeJaejun98 committed:
fix: adminPage()로 변경

LeeJaejun98 committed:
Merge remote-tracking branch 'origin/develop' into feature/front

LeeJaejun98 committed:
fix: adminController

LeeJaejun98 committed:
fix: function 재추가

LeeJaejun98 committed:
fix: login html 수정

LeeJaejun98 committed:
fix: email & nickname

LeeJaejun98 committed:
fix: getAll 삭제
